modules/oe_webtools_media/src/Plugin/Validation/Constraint/WebtoolsMediaConstraintValidator.php

```diff
@@ -46,7 +46,7 @@ public function validate($value, Constraint $constraint) {
     }
 
     // Add violation in case incorrect services.
-    if (!isset($snippet['service']) || $snippet['service'] !== $widget_types[$constraint->widgetType]['service']) {
+    if (!isset($snippet['service']) || !in_array($snippet['service'], $widget_types[$constraint->widgetType]['services'])) {
```

Comment:
Also BC issue here. Check for the presence of `services` and if not, case `service` to an array and use that. This is in case someone extended the source class.